modules/openapi-generator/src/main/resources/python-flask/requirements.mustache

```diff
@@ -1,7 +1,15 @@
 connexion >= 2.6.0; python_version>="3.6"
+# connexion 2.6 dropped support for python < 3.6
 connexion >= 2.3.0; python_version=="3.5"
 connexion >= 2.3.0; python_version=="3.4"
+{{#supportPython2}}
 connexion == 2.4.0; python_version<="2.7"
+{{/supportPython2}}
+# werkzeug is a dep of connexion. They need to record their dep.
```

Comment:
How about the simpler
```
# connexion requires werkzeug but connexion < 2.4.0 does not install werkzeug
# we must peg werkzeug versions below to fix connexion
# https://github.com/zalando/connexion/pull/1044
werkzeug == 0.16.1; python_version=="3.4"
werkzeug == 0.16.1; python_version=="3.5"
```
Note: for python versions ==2.7 and >= 3.6 we should not need to install werkzeug
because per https://github.com/zalando/connexion/commit/7111bc91ee9ffcdce1899c06205e61f634972228
this is fixed in connexion 2.4.0, and 2.6.0 which we are installing for those versions of python

Reply:
I applied your suggestion, plus I simplifeid it further using the or syntax from PEP 496: https://www.python.org/dev/peps/pep-0496/#examples